src/test/scala/org/camunda/feel/impl/interpreter/InterpreterUnaryTest.scala

```diff
@@ -185,12 +185,27 @@ class InterpreterUnaryTest
   it should "compare to a conjunction (and)" in {
     evalUnaryTests(true, "true and true") shouldBe ValBoolean(true)
     evalUnaryTests(true, "false and true") shouldBe ValBoolean(false)
+
+    evalUnaryTests(true, "true and null") shouldBe ValBoolean(false)
```

Comment:
❌ This case is wrong, the result should be `null` (there are more such incorrect cases added in this commit)

```
true and null
// null
```

See [docs](https://camunda.github.io/feel-scala/docs/1.14/reference/language-guide/feel-boolean-expressions/#conjunction--and)

Reply:
Sorry, but the cases are correct :sweat_smile: In the docs, it is about expressions but the test cases are about unary-tests :mag: 

The difference is that a unary-tests is compared to the input value.  So, `true and null` evaluates to `null`, and then it compares to `null` with `true` that result in `false`.

Reply:
Happy to chat about it :speaking_head:

Reply:
>So, true and null evaluates to null, and then it compares to null with true that result in false.

I see now how it works: the last comparison is due to it being a unary test. It compares the input (`true`) with the result of the expression `null`. These are not the same, so the result is `false`.

You could argue that this test does not verify the behavior of evaluating the unary test, e.g. `true and null` is also not equal to "random string". But you mentioned that verifying the evaluation was not the intent of this test case. The intent is to verify that it can be parsed (this was the bug). Users should generally not use this `true and null` expression as a unary test.